lib/Experian/IDAuth.pm

```diff
@@ -168,7 +163,7 @@ sub _build_request {
 sub _send_request {
     my $self = shift;
 
-    my $request = $self->{request_as_xml} || die 'needs request';
+    my $request = $self->{request_as_xml} // die 'needs request';
```

Comment:
Might need a `use 5.010` for this.  If there's no case that `$self->{request_as_xml} == undef`, perhaps just better to keep it on `||`.

Reply:
5.010 is sufficiently ancient these days that it would probably be fine just to

    use 5.010;

at the top